docs/zh/user-guide/chat/managing-tasks.md

```
@@ -25,18 +26,19 @@ sidebar_position: 5
 在 Wegent 中，对话是用户与 AI 智能体交互的基本方式。每次对话会创建一个 Task（任务），记录完整的对话历史。
 
 **核心概念**:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)